src/notebooks/visualization_notebook.ipynb

```diff
@@ -187,6 +187,52 @@
     "print('Posterior emissions :', total_posterior_emissions, 'Tg/y')"
    ]
   },
+  {
```

Comment:
can we make a separate PR for the sectoral emission attribution to avoid confusion?